resources/windows/wix/sequences.wxs.erb

```diff
@@ -14,7 +14,9 @@
         CMDLINE_INSTALLDIR
       </Custom>
       <Custom Action='SetIniPropertyValues' After='AppSearch' />
-      <Custom Action='GetLocalisedAccountNames' After='AppSearch' />
+      <Custom Action='GetLocalisedAccountNames' After='AppSearch'>
```

Comment:
What if we changed the `execute` type?
http://wixtoolset.org/documentation/manual/v3/xsd/wix/customaction.html